composer.json

```diff
@@ -28,19 +28,23 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "http-interop/http-middleware": "^0.4.1",
-        "laminas/laminas-mvc": "^3.1",
+        "php": "^7.3",
```

Comment:
I'm assuming 8.0 support is for a later PR?

Reply:
Yes. psalm and php 8 as separate PR